enterprise/admin/__init__.py

```diff
@@ -18,8 +18,9 @@
 from enterprise.django_compatibility import reverse
 from enterprise.lms_api import CourseApiClient, EnrollmentApiClient
 from enterprise.models import (  # pylint:disable=no-name-in-module
-    EnterpriseCustomer, EnterpriseCustomerUser, EnterpriseCustomerBrandingConfiguration,
-    EnterpriseCustomerIdentityProvider, HistoricalUserDataSharingConsentAudit, PendingEnterpriseCustomerUser,
+    EnrollmentNotificationEmailTemplate, EnterpriseCustomer, EnterpriseCustomerUser,
+    EnterpriseCustomerBrandingConfiguration, EnterpriseCustomerIdentityProvider,
+    HistoricalUserDataSharingConsentAudit, PendingEnterpriseCustomerUser,
```

Comment:
I absolutely agree, but it'd be impossible to make the `isort` linter accept that without making that change on all existing code - which might cause some issues to PRs in flight. I'd love to handle that as a separate PR though.